chpl/chpl-service/src/main/java/gov/healthit/chpl/domain/CertificationResultTestStandard.java

```diff
@@ -43,7 +43,7 @@
     /**
      * Name of test standard
      */
-    @XmlElement(required = true)
+    @XmlElement(required = false, nillable = true)
```

Comment:
Should this have been required = false all along?

Reply:
Same as above...